include/wlc/wlc.h

```diff
@@ -138,6 +137,14 @@ struct wlc_interface {
       WLC_NONULL void (*resolution)(wlc_handle output, const struct wlc_size *from, const struct wlc_size *to);
 
       struct {
+          /** Output context is created. This generally happens on startup and when current tty changes */
+          void (*created)(wlc_handle output);
+
+          /** Output context is destroyed */
+          void (*destroyed)(wlc_handle output);
+      } ctx;
```

Comment:
In general wlc hasn't use shortened words in the interface, so I would just make this "context"